frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/configure-step.tsx

```diff
@@ -26,7 +28,12 @@ export const Configure: React.FC<ConfigureProps> = ({ state, dispatch, mode }) =
 
   return (
     <Form noValidate={false}>
-      <EncryptionFormGroup state={state} dispatch={dispatch} mode={mode} />
+      <EncryptionFormGroup
+        state={state}
+        dispatch={dispatch}
+        mode={mode}
+        hasKmsSupport={isKmsSupported}
```

Comment:
I don't see the need to introduce a new prop here. Just use the flag inside EncryptionFormGroup.
```suggestion
```